build/Common.prod.props

```diff
@@ -67,6 +67,7 @@
     <PackageProjectUrl>https://opentelemetry.io</PackageProjectUrl>
     <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
     <Authors>OpenTelemetry Authors</Authors>
+    <Copyright>Copyright The OpenTelemetry Authors</Copyright>
```

Comment:
```suggestion
    <Copyright>Copyright the OpenTelemetry Authors</Copyright>
```

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/blob/2852b22bf8a953c19fabe8584e9a74304eb7fc6c/src/OpenTelemetry.Api/BaseProvider.cs#L2

Reply:
The current casing is correct. See https://github.com/open-telemetry/community/blob/main/CONTRIBUTING.md#copyright-notices